back/psychoApp/models/terapist.py

```diff
@@ -66,7 +66,7 @@ def create_superuser(
 # we can confirm that the profesional_registretion is real.
 
 
-class Terapist(AbstractBaseUser):
+class Therapist(AbstractBaseUser):
```

Comment:
Please change the meta with the correct spanish name `Psicologo`

Reply:
done with change in the model Therapist